local-tests/test_update.py

```diff
@@ -93,12 +93,12 @@ def query_runtime_version(nodes):
 print('Restarting remaining nodes with new binary')
 chain.stop(nodes=[0, 1, 2])
 chain.set_binary(newbin, nodes=[0, 1, 2])
-print('Waiting a bit')
-sleep(10)
-chain.start('new', nodes=[0, 1, 2])
-
 print('Waiting 30s')
 sleep(30)
+chain.start('new', nodes=[0, 1, 2])
+
+print('Waiting 90s')
+sleep(90)
 
```

Comment:
we could check finalization here (20 lines below) as well, couldn't we?

Reply:
We could, but currently finalization tends to be super flaky when starting small local chain all-nodes-at-once (sometimes it needs a few minutes to kick in). We really don't need another layer of flakiness in this test ^^'